### ISSUE TITLE
com.github.javaparser.ParseProblemException: Lexical error at line 1, column 1

### ISSUE THREAD

**Issue by yegor256** [volodya-lombrozo/jtcop#4]
I can't understand what is the reason for this error and I can't find the broken file:

```
[INFO] --- test-naming-conventions:0.1.2:check (check-tests-names) @ eo-maven-plugin ---
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for eo 1.0-SNAPSHOT:
[INFO]
[INFO] eo ................................................. SUCCESS [ 20.137 s]
[INFO] eo-parser .......................................... SUCCESS [ 34.644 s]
[INFO] eo-maven-plugin .................................... FAILURE [  1.049 s]
[INFO] eo-runtime ......................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  56.441 s
[INFO] Finished at: 2023-01-12T16:56:17+03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.volodya-lombrozo:test-naming-conventions:0.1.2:check (check-tests-names) on project eo-maven-plugin: com.github.javaparser.ParseProblemException: Lexical error at line 1, column 1.  Encountered: "\u0000" (0), after : ""
[ERROR] Problem stacktrace :
[ERROR]   com.github.javaparser.GeneratedJavaParserTokenManager.getNextToken(GeneratedJavaParserTokenManager.java:3054)
[ERROR]   com.github.javaparser.GeneratedJavaParser.jj_scan_token(GeneratedJavaParser.java:13593)
[ERROR]   com.github.javaparser.GeneratedJavaParser.jj_3_1(GeneratedJavaParser.java:9479)
[ERROR]   com.github.javaparser.GeneratedJavaParser.jj_2_1(GeneratedJavaParser.java:8346)
[ERROR]   com.github.javaparser.GeneratedJavaParser.CompilationUnit(GeneratedJavaParser.java:79)
[ERROR]   com.github.javaparser.JavaParser.parse(JavaParser.java:129)
[ERROR]   com.github.javaparser.JavaParser.parse(JavaParser.java:239)
[ERROR]   com.github.javaparser.StaticJavaParser.parse(StaticJavaParser.java:177)
[ERROR]   com.github.lombrozo.testnames.JavaTestCode.names(JavaTestCode.java:22)
[ERROR]   com.github.lombrozo.testnames.RuleForAllTests.validate(RuleForAllTests.java:15)
[ERROR]   com.github.lombrozo.testnames.ValidateMojo.execute(ValidateMojo.java:37)
[ERROR]   org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
[ERROR]   org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2(MojoExecutor.java:370)
[ERROR]   org.apache.maven.lifecycle.internal.MojoExecutor.doExecute(MojoExecutor.java:351)
[ERROR]   org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:215)
[ERROR]   org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:171)
[ERROR]   org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:163)
[ERROR]   org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
[ERROR]   org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
[ERROR]   org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
[ERROR]   org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
[ERROR]   org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:294)
[ERROR]   org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
[ERROR]   org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
[ERROR]   org.apache.maven.cli.MavenCli.execute(MavenCli.java:960)
[ERROR]   org.apache.maven.cli.MavenCli.doMain(MavenCli.java:293)
[ERROR]   org.apache.maven.cli.MavenCli.main(MavenCli.java:196)
[ERROR]   java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[ERROR]   java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[ERROR]   java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[ERROR]   java.base/java.lang.reflect.Method.invoke(Method.java:566)
[ERROR]   org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)
[ERROR]   org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)
[ERROR]   org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)
[ERROR]   org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)
[ERROR]
[ERROR] -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.github.volodya-lombrozo:test-naming-conventions:0.1.2:check (check-tests-names) on project eo-maven-plugin: com.github.javaparser.ParseProblemException: Lexical error at line 1, column 1.  Encountered: "\u0000" (0), after : ""
Problem stacktrace :
  com.github.javaparser.GeneratedJavaParserTokenManager.getNextToken(GeneratedJavaParserTokenManager.java:3054)
  com.github.javaparser.GeneratedJavaParser.jj_scan_token(GeneratedJavaParser.java:13593)
  com.github.javaparser.GeneratedJavaParser.jj_3_1(GeneratedJavaParser.java:9479)
  com.github.javaparser.GeneratedJavaParser.jj_2_1(GeneratedJavaParser.java:8346)
  com.github.javaparser.GeneratedJavaParser.CompilationUnit(GeneratedJavaParser.java:79)
  com.github.javaparser.JavaParser.parse(JavaParser.java:129)
  com.github.javaparser.JavaParser.parse(JavaParser.java:239)
  com.github.javaparser.StaticJavaParser.parse(StaticJavaParser.java:177)
  com.github.lombrozo.testnames.JavaTestCode.names(JavaTestCode.java:22)
  com.github.lombrozo.testnames.RuleForAllTests.validate(RuleForAllTests.java:15)
  com.github.lombrozo.testnames.ValidateMojo.execute(ValidateMojo.java:37)
  org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
  org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2(MojoExecutor.java:370)
  org.apache.maven.lifecycle.internal.MojoExecutor.doExecute(MojoExecutor.java:351)
  org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:215)
  org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:171)
  org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:163)
  org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
  org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
  org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
  org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
  org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:294)
  org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
  org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
  org.apache.maven.cli.MavenCli.execute(MavenCli.java:960)
  org.apache.maven.cli.MavenCli.doMain(MavenCli.java:293)
  org.apache.maven.cli.MavenCli.main(MavenCli.java:196)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:566)
  org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)
  org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)
  org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)
  org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)

    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:375)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:351)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:171)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:163)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:294)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:960)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: org.apache.maven.plugin.MojoExecutionException: com.github.javaparser.ParseProblemException: Lexical error at line 1, column 1.  Encountered: "\u0000" (0), after : ""
Problem stacktrace :
  com.github.javaparser.GeneratedJavaParserTokenManager.getNextToken(GeneratedJavaParserTokenManager.java:3054)
  com.github.javaparser.GeneratedJavaParser.jj_scan_token(GeneratedJavaParser.java:13593)
  com.github.javaparser.GeneratedJavaParser.jj_3_1(GeneratedJavaParser.java:9479)
  com.github.javaparser.GeneratedJavaParser.jj_2_1(GeneratedJavaParser.java:8346)
  com.github.javaparser.GeneratedJavaParser.CompilationUnit(GeneratedJavaParser.java:79)
  com.github.javaparser.JavaParser.parse(JavaParser.java:129)
  com.github.javaparser.JavaParser.parse(JavaParser.java:239)
  com.github.javaparser.StaticJavaParser.parse(StaticJavaParser.java:177)
  com.github.lombrozo.testnames.JavaTestCode.names(JavaTestCode.java:22)
  com.github.lombrozo.testnames.RuleForAllTests.validate(RuleForAllTests.java:15)
  com.github.lombrozo.testnames.ValidateMojo.execute(ValidateMojo.java:37)
  org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
  org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2(MojoExecutor.java:370)
  org.apache.maven.lifecycle.internal.MojoExecutor.doExecute(MojoExecutor.java:351)
  org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:215)
  org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:171)
  org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:163)
  org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
  org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
  org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
  org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
  org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:294)
  org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
  org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
  org.apache.maven.cli.MavenCli.execute(MavenCli.java:960)
  org.apache.maven.cli.MavenCli.doMain(MavenCli.java:293)
  org.apache.maven.cli.MavenCli.main(MavenCli.java:196)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:566)
  org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)
  org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)
  org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)
  org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)

    at com.github.lombrozo.testnames.ValidateMojo.execute (ValidateMojo.java:48)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:370)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:351)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:171)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:163)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:294)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:960)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: com.github.javaparser.ParseProblemException: Lexical error at line 1, column 1.  Encountered: "\u0000" (0), after : ""
Problem stacktrace :
  com.github.javaparser.GeneratedJavaParserTokenManager.getNextToken(GeneratedJavaParserTokenManager.java:3054)
  com.github.javaparser.GeneratedJavaParser.jj_scan_token(GeneratedJavaParser.java:13593)
  com.github.javaparser.GeneratedJavaParser.jj_3_1(GeneratedJavaParser.java:9479)
  com.github.javaparser.GeneratedJavaParser.jj_2_1(GeneratedJavaParser.java:8346)
  com.github.javaparser.GeneratedJavaParser.CompilationUnit(GeneratedJavaParser.java:79)
  com.github.javaparser.JavaParser.parse(JavaParser.java:129)
  com.github.javaparser.JavaParser.parse(JavaParser.java:239)
  com.github.javaparser.StaticJavaParser.parse(StaticJavaParser.java:177)
  com.github.lombrozo.testnames.JavaTestCode.names(JavaTestCode.java:22)
  com.github.lombrozo.testnames.RuleForAllTests.validate(RuleForAllTests.java:15)
  com.github.lombrozo.testnames.ValidateMojo.execute(ValidateMojo.java:37)
  org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
  org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2(MojoExecutor.java:370)
  org.apache.maven.lifecycle.internal.MojoExecutor.doExecute(MojoExecutor.java:351)
  org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:215)
  org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:171)
  org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:163)
  org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
  org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
  org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
  org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
  org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:294)
  org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
  org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
  org.apache.maven.cli.MavenCli.execute(MavenCli.java:960)
  org.apache.maven.cli.MavenCli.doMain(MavenCli.java:293)
  org.apache.maven.cli.MavenCli.main(MavenCli.java:196)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:566)
  org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)
  org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)
  org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)
  org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)

    at com.github.javaparser.StaticJavaParser.handleResult (StaticJavaParser.java:280)
    at com.github.javaparser.StaticJavaParser.parse (StaticJavaParser.java:177)
    at com.github.lombrozo.testnames.JavaTestCode.names (JavaTestCode.java:22)
    at com.github.lombrozo.testnames.RuleForAllTests.validate (RuleForAllTests.java:15)
    at com.github.lombrozo.testnames.ValidateMojo.execute (ValidateMojo.java:37)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:370)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:351)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:171)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:163)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:294)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:960)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
```

```
$ mvn -version
Apache Maven 3.8.6 (84538c9988a25aec085021c365c560670ad80f63)
Maven home: /usr/local/Cellar/maven/3.8.6/libexec
Java version: 11.0.16, vendor: Azul Systems, Inc., runtime: /Users/yegor/.sdkman/candidates/java/11.0.16-zulu/zulu-11.jdk/Contents/Home
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "11.7.1", arch: "x86_64", family: "mac"
```

**Comment by yegor256** [comment]
@volodya-lombrozo tested with Java 19, the same result:

```
$ mvn -version
Apache Maven 3.8.6 (84538c9988a25aec085021c365c560670ad80f63)
Maven home: /usr/local/Cellar/maven/3.8.6/libexec
Java version: 19.0.1, vendor: Azul Systems, Inc., runtime: /Users/yegor/.sdkman/candidates/java/19.0.1-zulu/zulu-19.jdk/Contents/Home
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "11.7.1", arch: "x86_64", family: "mac"
```

**Comment by volodya-lombrozo** [comment]
@yegor256 Can you run the next command, please:
```
mvn com.github.volodya-lombrozo:test-naming-conventions:0.1.4:check
```

**Comment by yegor256** [comment]
@volodya-lombrozo I just did, the same error:

```
$ mvn com.github.volodya-lombrozo:test-naming-conventions:0.1.4:check -pl :eo-maven-plugin
[INFO] Scanning for projects...
Downloading from oss.sonatype.org: https://oss.sonatype.org/content/groups/public/com/github/volodya-lombrozo/test-naming-conventions/0.1.4/test-naming-conventions-0.1.4.pom
Downloaded from oss.sonatype.org: https://oss.sonatype.org/content/groups/public/com/github/volodya-lombrozo/test-naming-conventions/0.1.4/test-naming-conventions-0.1.4.pom (6.9 kB at 3.5 kB/s)
Downloading from oss.sonatype.org: https://oss.sonatype.org/content/groups/public/com/github/volodya-lombrozo/test-naming-conventions/0.1.4/test-naming-conventions-0.1.4.jar
Downloaded from oss.sonatype.org: https://oss.sonatype.org/content/groups/public/com/github/volodya-lombrozo/test-naming-conventions/0.1.4/test-naming-conventions-0.1.4.jar (16 kB at 20 kB/s)
[INFO]
[INFO] ---------------------< org.eolang:eo-maven-plugin >---------------------
[INFO] Building eo-maven-plugin 1.0-SNAPSHOT
[INFO] ----------------------------[ maven-plugin ]----------------------------
[INFO]
[INFO] --- test-naming-conventions:0.1.4:check (default-cli) @ eo-maven-plugin ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  3.778 s
[INFO] Finished at: 2023-01-13T09:05:56+03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.volodya-lombrozo:test-naming-conventions:0.1.4:check (default-cli) on project eo-maven-plugin: java.lang.IllegalStateException: Failed to parse Java class by path /Volumes/ssd2/code/objectionary/eo/eo-maven-plugin/src/test/java/org/eolang/.DS_Store: Lexical error at line 1, column 1.  Encountered: "\u0000" (0), after : ""
[ERROR] Problem stacktrace :
[ERROR]   com.github.javaparser.GeneratedJavaParserTokenManager.getNextToken(GeneratedJavaParserTokenManager.java:3054)
[ERROR]   com.github.javaparser.GeneratedJavaParser.jj_scan_token(GeneratedJavaParser.java:13593)
[ERROR]   com.github.javaparser.GeneratedJavaParser.jj_3_1(GeneratedJavaParser.java:9479)
[ERROR]   com.github.javaparser.GeneratedJavaParser.jj_2_1(GeneratedJavaParser.java:8346)
[ERROR]   com.github.javaparser.GeneratedJavaParser.CompilationUnit(GeneratedJavaParser.java:79)
[ERROR]   com.github.javaparser.JavaParser.parse(JavaParser.java:129)
[ERROR]   com.github.javaparser.JavaParser.parse(JavaParser.java:239)
[ERROR]   com.github.javaparser.StaticJavaParser.parse(StaticJavaParser.java:177)
[ERROR]   com.github.lombrozo.testnames.JavaTestCode.names(JavaTestCode.java:23)
[ERROR]   com.github.lombrozo.testnames.RuleForAllTests.validate(RuleForAllTests.java:15)
[ERROR]   com.github.lombrozo.testnames.ValidateMojo.execute(ValidateMojo.java:37)
[ERROR]   org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
[ERROR]   org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2(MojoExecutor.java:370)
[ERROR]   org.apache.maven.lifecycle.internal.MojoExecutor.doExecute(MojoExecutor.java:351)
[ERROR]   org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:215)
[ERROR]   org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:171)
[ERROR]   org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:163)
[ERROR]   org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
[ERROR]   org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
[ERROR]   org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
[ERROR]   org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
[ERROR]   org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:294)
[ERROR]   org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
[ERROR]   org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
[ERROR]   org.apache.maven.cli.MavenCli.execute(MavenCli.java:960)
[ERROR]   org.apache.maven.cli.MavenCli.doMain(MavenCli.java:293)
[ERROR]   org.apache.maven.cli.MavenCli.main(MavenCli.java:196)
[ERROR]   java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
[ERROR]   java.base/java.lang.reflect.Method.invoke(Method.java:578)
[ERROR]   org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)
[ERROR]   org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)
[ERROR]   org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)
[ERROR]   org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

**Comment by yegor256** [comment]
@volodya-lombrozo I think it's a bad idea to think that `.DS_Store` is a Java class :)

**Comment by volodya-lombrozo** [comment]
@yegor256 Apparently, yes :)

**Comment by volodya-lombrozo** [comment]
@yegor256 I've solved that problem in the `0.1.5` version. Could you please run the next command again, please?
```
mvn com.github.volodya-lombrozo:test-naming-conventions:0.1.5:check
```

**Comment by yegor256** [comment]
@volodya-lombrozo now it's good
